test/resources/streams.js

```diff
@@ -46,6 +46,9 @@ describe('Streams', () => {
     common.restoreStubs(stubs);
   });
   it('should fetch subscriptions', () => {
+    const params = {
+      subscriptions: JSON.stringify([{ name: 'off topic' }]),
+    };
     const validator = (url, options) => {
```

Comment:
I would recommend adding the parameter check in the request validator as well, like as follows (copied from another PR).

```
const validator = (url, options) => {
      url.should.contain(`${common.config.apiURL}/get_stream_id`);
      options.should.not.have.property('body');
      const urldata = url.split('?', 2)[1].split('&'); // URL: host/get_stream_id?key=value&key=value...
      urldata.length.should.equal(1);
      urldata.should.contain(`stream=${params.stream}`);
    };
```

Reply:
@aero31aero Thanks for the review. updated!